plasmapy/physics/tests/test_distribution.py

```diff
@@ -0,0 +1,35 @@
+"""Tests for functions that uses Distribution functions."""
+
+import numpy as np
+import pytest
+from astropy import units as u
+
+
+from ...constants import (m_p, m_e, c, mu0, k_B, e, eps0, pi, e)
+from ..atomic import (ion_mass, charge_state)
+from ..distribution import (Maxwellian_1D)
+
+T_e = 30000*u.K;
+V_drift = 10*u.m/u.s;
+
+start = -5000;
+stop = - start;
+dv = 10000 * u.m/u.s;
+
+v_vect = np.arange(start,stop,dtype='float64')* dv;
+
+def test_Maxwellian_1D():
+    """test the 1D maxwellian distribution function"""
+
+    assert np.isclose(v_vect[Maxwellian_1D(v_vect,T=T_e, part='e',V_drift = 0*u.m/u.s ).argmax()].value,0.0)
+
+    assert np.isclose(v_vect[Maxwellian_1D(v_vect,T=T_e, part='e',V_drift = V_drift ).argmax()].value,V_drift.value)
+
+    #integral of the distribution over v_vect
+    integral = (Maxwellian_1D(v_vect,T= 30000*u.K, part='e')).sum()*dv
```

Comment:
If we used `scipy.integrate` here (I don't think adding scipy to requirements is an issue given we're working within the scientific Python stack), we'd also get an error estimate on the integral we could use for the next line!

Reply:
I'n not so used to Scipy, I didn't knew we could do it !  I'll check it out.